.github/workflows/cmake.yml

```diff
@@ -240,6 +245,11 @@ jobs:
 
     - name: Install pip packages
       run: pip install -r third_party/requirements.txt
+
+    - name: Install apt packages
```

Comment:
this job is run on our HW - it should not be needed here

Reply:
Done.